### ISSUE TITLE
top 5 nodejs packages that every developer should know

### ISSUE THREAD

**Issue by Worawat101** [section-io/engineering-education#461]
### Brief Summary: 
Before starting any project one should know about the development environment, workflows, dependencies. So, the work becomes easier. Now, there are some packages for NodeJS that simplify the overall workflow while developing a Node project and increases productivity. Many developers are unaware of the existence of these packages. The package mentioned below can help improve the overall development workflow.

### Key Takeaways: 
The packages help a lot in providing the required functionality without having to spend a lot of time implementing the functionality ourselves. After going through the article, the reader will know some of the distinct packages that will help save time, effort as well as improve the workflow. The package like Nodemon provides the hot reload feature, chalk package can improve the terminal readability, debug can improve the debugger experience, bluebird helps to deal with unavoidable and asynchronous processes, and faker helps to generate dummy data.

1. nodemon
2. chalk
3. debug
4. bluebird
5. faker

**Comment by ninjaginja** [comment]
@Worawat101 Thanks for the topic suggestion. We want to ensure that you're not simply rewriting what is already covered in docs. Can you please expand on how you intend to make this content unique? (Also, just a note to be mindful of grammar and readability. Some of the phrasing in the paragraph that you've provided doesn't make sense. We don't provide editing services, so we'll request that you put your article through a thorough proofing process prior to submitting.)

**Comment by Worawat101** [comment]
@ninjaginja I try to change the topic. think will better than old

**Comment by ninjaginja** [comment]
Closing due to inactivity. Please request to reopen if you are still working on this content. Or, if there is an open PR, please be sure to link it to this issue.
